pre-cache/check_space

```diff
@@ -11,8 +11,25 @@ validate_disk_space() {
     SERVICEACCOUNT=/var/run/secrets/kubernetes.io/serviceaccount
```

Comment:
As a couple of general notes:
- The first line of this file is odd. Is the ./ a typo or some syntax I'm not familiar with?
- This script and a few others in this dir do not have a .sh extension. This means they're bypassing the bash linters that are run as part of the CI - bashate and shellcheck. This probably needs to be a separate action to fix this and address all the warnings

Reply:
@sudomakeinstall2 Can you raise a story to capture Don's second point?

Reply:
https://issues.redhat.com/browse/CNF-9789